spec/menu_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'menu'
+
+describe Menu do
```

Comment:
One thing you aren't testing yet, is that you can initialise an instance with a custom menu

Reply:
Test added to check this.